### DISCUSSION TITLE
ABUSE REPORT

### DISCUSSION THREAD

**Discussion by Te44ri** [community/community#131051] [Other Feature Feedback, Questions, & Ideas] [answered]
This https://github.com/juzeon/fast-mail-bomber is SPAM tool that uses many API without any permission to SPAM victim. It is literally written in description!

please block!

**Comment by Daeraxa** [comment]
1) You posted this in the wrong cateogry
2) So what? It is a tool that can have legitimate uses just like any other. There is no need to take it down.

**Comment by israeloliveiradev** [comment]
?

**Comment by Akash1134** [comment, marked as answer]
Hi @Te44ri , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by OCEANOFANYTHINGOFFICIAL** [comment]
GitHub is an open space for all coders to host any kind of project. If you are facing any harm from the project or any other user of the project, you may report it to the authority.
In this case, we can let the project as a gun. suppose someone is harassing you with a gun. Now as a responsible human being you may report to the user of the gun, not to the creator of the gun. I hope it is clear now, appreciate your personal rights and I'm deeply apologize for your inconvenience from the project
